default.css

```diff
@@ -763,10 +786,32 @@
 		{
 			display: none;
 		}
-        
-		#footer
+		
+		#menu
 		{
-			display: none;
+			/* Don't constrain menu height on mobile */
+			height: initial;
+		}
+		
+		#menu ul
+		{
+			/* Vertical menu on mobile */
+			flex-direction: column;
+			justify-content: center;
+		}
+		
+		#menu li a, #menu li span
+		{
+			/* Make nav buttons take up more room on mobile */
+			width: 100%;
+			/* Include padding in the width */
+			box-sizing: border-box;

```

Comment:
[`border-box`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) is really nice - it makes padding be taken into account when setting widths. Without `border-box`, if you have an element with 10px padding and 100% width inside a 100px container, it would end up being 120px, which is probably not what you want. With border-box, it would be 100px, with 80px of content inside.

These days, it's pretty common to just set 
```css
* {
box-sizing: border-box;
}
```
for your whole site.